pom.xml

```diff
@@ -119,7 +123,18 @@
 					<target>${version.java}</target>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-surefire-plugin</artifactId>
+			    <version>2.19.1</version>
```

Comment:
please use a property for the version of this plugin

Reply:
Done.